### ISSUE TITLE
Review of Poster by DataWaves Team

### ISSUE THREAD

**Issue by libin-n-g** [INFO-526-FA24/project-2-stat-squad#7]
### 1. What is the good thing about the poster?  

- Lots of plots, and good segregation of different Gradient decent. Good use of 3D plot. 

### 2. What do you think can/should be improved?  

- The animations (for neural networks and 3D plots of gradient descent) might be lost when printing the poster. It would be best if you fixed the most effective frame from gifs while printing. 
- The visualizations seem similar for different plots. You can add an error term while generating the data to increase uncertainty and, the extent of error in the data might have a surprising effect on how different gradient descent differs. 
- In the last plot, it might be better to use different color schemes for plotting different optimizers. You should combine both shape and color legends as it seems to show the same data. 
- The equations are confusing as there do not seem to be explanations for each parameter like η, θ, etc. Only f(x,y) seems to be intuitive enough to be understood from the plot. 

---

@INFO-526-FA24/datawaves **conducted this review**

**Comment by PrasanthGubbala** [comment]
The feedback has been reviewed, and the necessary improvements (animation frame for printing, adding error terms, adjusting visualizations, and clarifying equations) will be addressed accordingly to finalize the poster.
